### DISCUSSION TITLE
Why can't the first semester be Spring?

### DISCUSSION THREAD

**Discussion by duchuynh** [qu8n/pennplanner#3] [Q&A] [answered]
For example, I noticed that even though I am starting in Spring 2024, I can only select Fall 2023 and there is no option to select Spring 2024. Perhaps this can be added.

**Comment by qu8n** [comment, marked as answer]
Hi @duchuynh, great question, and welcome to the program!

PennPlanner is designed to organize semesters by academic year, which in the US is typically a set of Fall, Spring, and Summer semesters. As a result, because you're starting in Spring 2024, the first semester in your first academic year will be Fall 2023. 

In your planner, feel free to ignore Fall 2023 and drop your first course(s) into Spring 2024.

**Reply by duchuynh** [reply]
Sounds good. Thanks!
